### DISCUSSION TITLE
Welcome to krabs Discussions!

### DISCUSSION THREAD

**Discussion by micheleriva** [micheleriva/krabs#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GodGardensGuns** [comment]
Hello, my name is Liam and I currently use Next.js on 3 work projects (and a number of other personal ones). Just wanted to stop by to say that I am very interested in this project, as it has an immediate use case in one of my work projects. The only issue is that SSG is not yet supported. Once it is, there's a very good chance my team will switch to using this library.

Although I don't have much free time for open source contribution these days, I will be keeping an eye on this repo and contribute however I can when time/circumstances allow. I look forward to seeing where this project goes!

**Reply by micheleriva** [reply]
Hi @dayvista!
Thank you for your interest in Krabs.

If you really need SSG + ISR, there's a quick solution for that: you can use a CDN as a caching layer. That way, Cloudflare, Akamai, Fastly, or any other CDN will cache your pages for `x` minutes and serve the cached version until a given TTL allows it. Does it make sense? 🙂

**Reply by GodGardensGuns** [reply]
Absolutely! Thanks. We'll look into that ASAP.

**Reply by micheleriva** [reply]
@dayvista let me know how it goes! :D
